### PR TITLE
Fix testIncrementalHandOffReadsThroughEndOffsets in Kafka/KinesisIndexTaskTest

### PR DESCRIPTION
**Pull request by jihoonson**
Both tests are running with transactional mode, so offsets might not be consecutive. This PR runs two tasks to simulate stale replica.

### CODE REVIEW

File: extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java [hunk 1]
```diff
@@ -928,34 +929,68 @@ public void testIncrementalHandOffReadsThroughEndOffsets() throws Exception
...
       Thread.sleep(10);
     }
-    Map<Integer, Long> currentOffsets = ImmutableMap.copyOf(task.getRunner().getCurrentOffsets());
+    Map<Integer, Long> currentOffsets = ImmutableMap.copyOf(normalReplica.getRunner().getCurrentOffsets());
     Assert.assertEquals(checkpoint1.getPartitionSequenceNumberMap(), currentOffsets);
 
-    // Simulating the case when another replica has consumed up to the offset of 8
-    task.getRunner().setEndOffsets(ImmutableMap.of(0, 8L), false);
+    normalReplica.getRunner().setEndOffsets(currentOffsets, false);
+    staleReplica.getRunner().setEndOffsets(currentOffsets, false);
```
Comment (clintropolis) [inline extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java]: Is it assumed that `staleReplica` has offsets behind `normalReplica` offsets? Is this testable?

File: indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java [hunk 2]
```diff
@@ -534,6 +534,7 @@ public void run()
                 } else {
                   rows = new ArrayList<>();
                   for (byte[] valueBytes : valueBytess) {
+                    log.info("valueBytes len: " + valueBytes.length);
```
Comment (clintropolis) [inline indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java]: Is this really an `info` level log?

Comment (jihoonson) [reply indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java]: Oh, it was my debugging log. Will remove.

File: extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java (hunk 1 above)
Comment (jihoonson) [reply extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java]: That is my intention, but this test doesn't guarantee that staleReplica is always behind normalReplica. My assumption here is that staleReplica would be stale in most runs.

**Pull request merged**